bigquery/views/job-alert.sql

```diff
@@ -2,6 +2,14 @@ SELECT
   id,
   CAST(created_at AS date) AS created_date,
   CAST(updated_at AS date) AS updated_date,
+  active,
+  CAST(unsubscribed_at AS date) AS unsubscribed_date,
```

Comment:
This is a datetime - don't know if that matters.

Reply:
Yes, the conversion to a date is deliberate - because downstream this is used to calculate loads of counts by day, and it saves a lot of duplicate type conversion to put it here.